etc/profile-a-l/chafa.profile

```diff
@@ -0,0 +1,56 @@
+# Firejail profile for chafa
+# Description: chafa is an image viewer.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include chafa.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+# Users may want to view images in ${HOME}
+#include disable-xdg.inc
+
+# Users may want to view images in ${HOME}
+#include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+# Users may want to view images in /usr/share
+#include whitelist-usr-share-common.inc
```

Comment:
Can you check for chafa files in /usr/share and change the comment as follow.
```suggestion
# Add the folloing to your chafa.local if you do not need to view images in /usr/share.
#whitelist /usr/share/chafa  <-- or whatever is required
#include whitelist-usr-share-common.inc
```

Reply:
I'm a bit confused. Wouldn't you `whitelist /usr/share/chafa` if you *do* need to view images in /usr/share?

Regardless, chafa does not create a `/usr/share/chafa`, so I don't think this is needed.

Reply:
> chafa does not create a /usr/share/chafa

Of course not. It would mean a normal user can write in `/usr/share`.

The (possible) files there are created by the package and read by chafa, so check either one.

Reply:
> Wouldn't you whitelist /usr/share/chafa if you do need to view images in /usr/share?

No, it's commented. It is an opt-in for users to harden their profile if they do not need to view images in /usr/share`

Reply:
> The (possible) files there are created by the package and read by chafa, so check either one.
This is what I meant. The package (at least by Arch) does not create a `/usr/share/chafa`. It does create a `/usr/share/gtk-doc/html/chafa`, but that directory is meant to be looked at with a web browser and not an image viewer.

Should I add the line anyway? Is it preferable to:
```
#blacklist /usr/share
```

Reply:
1. No.
2. No. It will block access to dirs like `/usr/share/terminfo` and likely break.